.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Hello @${{ github.actor }} ,Thank you for raising an issue. Please be detailed and clear along with an acceptance criteria.'
+        pr-message: 'Hello @${{ github.actor }} ,Thank you for submitting a Pull Request. Make sure you have followed all the contribution guidelines.'
```

Comment:
Sorry man, another suggestion. Please.
```suggestion
        issue-message: 'Hello @${{ github.actor }}, thank you for raising an issue. Please be detailed and clear along with an acceptance criteria.'
        pr-message: 'Hello @${{ github.actor }}, thank you for submitting a Pull Request. Make sure you have followed all the contribution guidelines.'
```

Reply:
Updated @praveenscience . It's okay, no sorry ,instead points maybe? Haha !